ui/scss/component/_file-render.scss

```diff
@@ -35,10 +35,6 @@
     height: 100%;
     object-fit: contain;
   }
-
-  video {
```

Comment:
we want to keep this style for non-fullscreen. Maybe you could change it do `default` when it's full-screen?

Reply:
Changed cursor to none on the following conditions
```
.vjs-fullscreen.vjs-user-inactive.vjs-playing
```